robottelo.properties.sample

```diff
@@ -127,3 +127,13 @@ test.virtual_machines_list=127.0.0.1,127.0.0.1,127.0.0.1,127.0.0.1,127.0.0.1
 # Class `ConcurrentTestCase` and its subclasses use this setting when
 # computing statistics of each performance test case, grouped in buckets.
 csv.num_buckets=10
+
+# Target repository names to be synchronized by Pulp.
+test.target_repos=Repo Example 1,Repo-Example-2
```

Comment:
@Ichimonji10, I actually read this entry here: https://github.com/danuzclaudes/robottelo/commit/b97b2635be1d1caf7105657664dd9b57bd582a72#diff-30051cc3cf62fb3c2573508e5d08d99cR88
But as you've said, I'd raise exception if the field is empty. BTW, if the user provides invalid repository name, the synchronization would just ignore it and continue next.